export_variables.sh

```diff
@@ -26,6 +26,11 @@ case "$test_case" in
     tmt_plan_suffix="-openshift-4"
     test_name="test-openshift-4"
     ;;
+  "pytest")
+    context_suffix=" - OpenShift 4"
```

Comment:
Don't we want to differentiate openshift4 tests and openshift 4 pytest tests with the context_suffix?

Reply:
In GitHub action there will be defined `user_context`. I will file a PR to httpd-container. Good point.

Reply:
https://github.com/sclorg/httpd-container/pull/212

Reply:
After long time, you are right. It will be easy to maintain it. https://github.com/sclorg/tfaga-wrapper/pull/24/commits/ad6a955d618920e4f7eefb4ae5ca0398e81242c3